draft-ietf-tls-sni-encryption.md

```diff
@@ -266,9 +266,9 @@ MITM attack is troubling.
 There are other classes of solutions in which the master secret is verified by
 verifying a certificate provided by the protected service. These solutions offer
 more protection against a Man-In-The-Middle attack by the fronting service. The
-downside is the the client will not verify the identity of the fronting service
-with risks discussed in {#frontingspoofing}, but solutions will have to
-mitigate this risks. Overall, end-to-end TLS to the protected service is preferable.
+downside is the the client will not verify the identity of the fronting service,
```

Comment:
I guess we can fix the "the the" while we're here, even though it's preexisting.  ("that the" is intended, it looks like.)
The actual content change here looks good, though.

Reply:
OK, fixing that too. I am still not entirely happy with how the last sentence concludes it. Something like "Overall, end-to-end TLS to the protected service is preferable, but it is important to also provides way to authenticate the fronting service." The proposed ESNI solution actually does that by verifying possession of the static DH half, so this is definitely possible.

Reply:
I'd be fine with your proposal above (nit: "provides a way")